### PR TITLE
fix(RouterStore): support objects for options of StoreRouterConnectingModule

### PR DESCRIPTION
**Pull request by fredgate**
Fix issue #573

### CODE REVIEW

File: modules/router-store/src/router_store_module.ts [hunk 1]
```diff
@@ -125,10 +125,12 @@ export const ROUTER_CONFIG = new InjectionToken(
 export const DEFAULT_ROUTER_FEATURENAME = 'routerReducer';
 
 export function _createDefaultRouterConfig(config: any): StoreRouterConfig {
```
Comment (brandonroberts) [inline modules/router-store/src/router_store_module.ts]: Change `config` argument type to `StoreRouterConfig | StoreRouterConfigFunction`

File: modules/router-store/src/router_store_module.ts [hunk 2]
```diff
@@ -125,10 +125,12 @@ export const ROUTER_CONFIG = new InjectionToken(
...
-  let _config = {};
+  let _config: any;
```
Comment (brandonroberts) [inline modules/router-store/src/router_store_module.ts]: Change `_config` to `StoreRouterConfig`

**Comment by brandonroberts** [comment]
Thanks @fredgate for the PR! I left you some feedback.

File: modules/router-store/src/router_store_module.ts (hunk 1 above)
Comment (fredgate) [reply modules/router-store/src/router_store_module.ts]: I agree

File: modules/router-store/src/router_store_module.ts (hunk 2 above)
Comment (fredgate) [reply modules/router-store/src/router_store_module.ts]: It makes sense

**Comment by brandonroberts** [comment]
Thanks!

**Pull request merged**